src/index.html

```diff
@@ -11,7 +11,190 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="get">
```

Comment:
Checklist point 1

Reply:
Think about the method as well

Reply:
Right, it is now fixed. I'll post it now.